internal/lightcone/hunt/returntodarkness/returntodarkness.go

```diff
@@ -0,0 +1,68 @@
+package returntodarkness
+
+import (
+	"math/rand"
+
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	ReturntoDarkness key.Modifier = "return_to_darkness"
+)
+
+var wasTriggered bool = false
```

Comment:
`wasTriggered` needs to be part of the modifier state. With it global, it will be shared between multiple characters that use this LC (+ multiple iterations of the sim running in parallel)

When adding as part of the state, you'll need to pass the state as a pointer reference so you can edit the `wasTriggered` bool. Meshing Cogs LC is an example of this: https://github.com/simimpact/srsim/blob/f1472e5630ca21889a927ca6bdd4ade8cb0afb25/internal/lightcone/harmony/meshingcogs/meshingcogs.go#L36-L54